src/picongpu/include/particles/ParticlesFunctors.hpp

```diff
@@ -313,6 +318,52 @@ struct CallIonization
 
 }; // struct CallIonization
 
-} //namespace particles
+/** Handles the synchrotron radiation emission of photons from electrons
+ *
+ * \tparam T_SpeciesName name of electron species
+ */
+template<typename T_SpeciesName>
+struct CallSynchrotronPhotons
+{
+    typedef T_SpeciesName SpeciesName;
+    typedef typename SpeciesName::type SpeciesType;
+    typedef typename SpeciesType::FrameType FrameType;
+    /* SelectedPhotonCreator will be either PhotonCreator or fallback: CreatorBase */
+    typedef typename traits::GetPhotonCreator<SpeciesType>::type SelectedPhotonCreator;
+
+    /** Functor implementation
+     *
+     * \tparam T_StorageStuple contains info about the particle species
+     * \tparam T_CellDescription contains the number of blocks and blocksize
+     *                           that is later passed to the kernel
+     * \param tuple An n-tuple containing the type-info of multiple particle species
+     * \param cellDesc points to logical block information like dimension and cell sizes
+     * \param currentStep The current time step
+     */
+    template<typename T_StorageTuple, typename T_CellDescription>
+    HINLINE void operator()(
+                            T_StorageTuple& tuple,
+                            T_CellDescription* cellDesc,
+                            const uint32_t currentStep,
+                            const synchrotronPhotons::SynchrotronFunctions& synchrotronFunctions) const
```

Comment:
The last parameter of this method is not mentioned/described in the doxygen description. Please add a description.